packages/react-resizin/src/hooks/useImage.ts

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Options } from 'resizin';
+
+import ResizinContext from '../components/ResizinContext';
+
+const useImage = (imageId: string, options: Options): string => {
+    const buildImageUrl = React.useContext(ResizinContext);
```

Comment:
It is usual to check if the component or the hook is used out of the provider of the context and if it is, then to log some error or warnin.g

Reply:
Great idea, adde (inspired by react-redux) along with hook tests 👍